src/fountain.rs

```diff
@@ -599,13 +601,26 @@ impl Part {
     }
 }
 
+/// Calculates the quotient of `a` and `b`, rounding the results towards
+/// positive infinity.
+///
+/// Note: there's an implementation on the `usize` type of this function,
+/// but it's not stable yet.
+#[must_use]
+fn div_ceil(a: usize, b: usize) -> usize {
+    let d = a / b;
+    let r = a % b;
+    if r > 0 && b > 0 {
```

Comment:
Line 611 will already panic if `b` is zero, i.e. this can be removed. Maybe let's add an `assert!(b>0)` at the start of the function?

Reply:
Fixed.

Reply:
Did you forget the `assert!`?

Reply:
@dspicher It will panic for the division by zero already so I didn't add it

Reply:
Prefer it to be a panic by division by zero since it's a division operation

Reply:
Alright.